e2e/process-services-cloud/process-custom-filters.e2e.ts

```diff
@@ -147,6 +261,184 @@ describe('Process list cloud', () => {
+            processCloudDemoPage.processListCloudComponent().getAllRowsStartDateColumn().then(function (list) {
+                const initialList = list.slice(0);
+                list.sort(function (firstStr, secondStr) {
+                    return firstStr.localeCompare(secondStr);
+                });
+                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+            });
+
+            processCloudDemoPage.editProcessFilterCloudComponent().setOrderFilterDropDown('DESC');
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkSpinnerIsDisplayed().checkSpinnerIsNotDisplayed();
+            processCloudDemoPage.processListCloudComponent().getAllRowsStartDateColumn().then(function (list) {
+                const initialList = list.slice(0);
+                list.sort(function (firstStr, secondStr) {
+                    return firstStr.localeCompare(secondStr);
+                });
+                list.reverse();
+                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+            });
+        });
+
+        it('[C305054] Should display processes ordered by initiator date when Initiator is selected from sort dropdown', async () => {
+            processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader().setStatusFilterDropDown('ALL')
+                .setSortFilterDropDown('Initiator').setOrderFilterDropDown('ASC');
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkSpinnerIsDisplayed().checkSpinnerIsNotDisplayed();
+            processCloudDemoPage.processListCloudComponent().getAllRowsInitiatorColumn().then(function (list) {
+                const initialList = list.slice(0);
+                list.sort(function (firstStr, secondStr) {
+                    return firstStr.localeCompare(secondStr);
+                });
+                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+            });
+
+            processCloudDemoPage.editProcessFilterCloudComponent().setOrderFilterDropDown('DESC');
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkSpinnerIsDisplayed().checkSpinnerIsNotDisplayed();
+            processCloudDemoPage.processListCloudComponent().getAllRowsInitiatorColumn().then(function (list) {
+                const initialList = list.slice(0);
+                list.sort(function (firstStr, secondStr) {
+                    return firstStr.localeCompare(secondStr);
+                });
+                list.reverse();
+                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+            });
+        });
+
+        it('[C305054] Should display processes ordered by processdefinitionid date when ProcessDefinitionId is selected from sort dropdown', async () => {
```

Comment:
To be sure that the list is sorted by processDefinitionId, i suggest to have in the preconditions processes created using at least two different process definitions. Currently one one process definition is used.

Reply:
good point, I will have a look into that.

Reply:
adding 2 Process Definitions to candidateuserapp which will help in testing the sorting, asc and desc, for  properties both 'processDefinitionId' and 'processDefinitionKey'.